src/Raven.Studio/typescript/viewmodels/database/tasks/exportDatabase.ts

```diff
@@ -135,17 +161,48 @@ class exportDatabase extends viewModelBase {
             if (!args.TransformScript) {
                 delete args.TransformScript;
             }
+
             const fileName = args.FileName;
             delete args.FileName;
             const json = JSON.stringify(args);
 
-            return 'curl -o "' + fileName + '.ravendbdump" --data "DownloadOptions=' + encodeURIComponent(json) + '" ' +
-                appUrl.forServer() + appUrl.forDatabaseQuery(db) + endpoints.databases.smuggler.smugglerExport;
+            return `curl.exe -o "${fileName}.ravendbdump" --data DownloadOptions="${json.replace(/"/g, '\\"')}" ${commandEndpointUrl(db)}`;
         });
-    }
 
-    copyCommandToClipboard() {
-        copyToClipboard.copy(this.exportCommand(), "Command was copied to clipboard.");
+        this.exportCommandBash = ko.pureComputed<string>(() => {
+            const db = this.activeDatabase();
+            if (!db) {
+                return "";
+            }
+
+            const args = this.model.toDto();
+            if (!args.TransformScript) {
+                delete args.TransformScript;
+            }
+            
+            const fileName = args.FileName;
+            delete args.FileName;
+            const json = JSON.stringify(args);
+
+            return `curl -o '${fileName}.ravendbdump' --data DownloadOptions='${json}' ${commandEndpointUrl(db)}`;
+        });
+
+        this.effectiveCommandLabel = ko.pureComputed(() => {
```

Comment:
remove this and simply use: getCommandTypeLabel(this.effectiveCommand) in view.

Reply:
done